packages/flutter_localizations/lib/src/l10n/material_ko.arb

```diff
@@ -0,0 +1,34 @@
+{
+  "scriptCategory": "dense",
```

Comment:
You can double-check the script categories on this page: https://material.io/guidelines/style/typography.html#typography-language-categories-reference

Reply:
I've checked them and changed Thai (th) to 'tall'.